utils.py

```diff
@@ -0,0 +1,86 @@
+import os
+from presidio_analyzer import AnalyzerEngine, PatternRecognizer
+from presidio_anonymizer import AnonymizerEngine
+from presidio_anonymizer.entities import OperatorConfig
+import json
+from pprint import pprint
+from openai import OpenAI
+
+client = OpenAI(api_key=os.getenv("OPENAI_KEY"))
```

Comment:
We should have a better way to a) request permission from the user to do this, and b) give users the configuration option to change this.